september-util/src/main/java/edu/asu/diging/gilesecosystem/septemberutil/service/impl/SystemMessageHandler.java

```diff
@@ -49,7 +49,7 @@ public SystemMessageHandler(String applicationId) {
     }
 
     @Override
-    public void handleError(String msg, Exception exception) {
+    public void handleException(String msg, Exception exception, String messageType) {
```

Comment:
yes, I'd create an extra enum. SystemMessageRequest is in the request package. In theory an app could use the the september util package but not the request package (if it's not hooked up to the kafka pipeline). In this case, it would have to have a dependency just for the error messages. You can map the enum to SystemMessageRequest in the enum, and then use it's just a method call to get from the enum to the string.